site/docs/v1/tech/apis/system.adoc

```diff
@@ -286,21 +286,21 @@ The Status API is used to retrieve the current status and metrics for FusionAuth
 [NOTE]
 ====
 FusionAuth also supports a system status check using Prometheus.
-// TBD add link when prometheus docs are live. See Prometheus documentation here.
+// TBD add link when Prometheus docs are live. See Prometheus documentation here.
```

Comment:
It's live, lets add it. https://fusionauth.io/docs/v1/tech/tutorials/prometheus/